src/actor/actor-sheet.js

```diff
@@ -13,6 +14,15 @@ export class ForbiddenLandsActorSheet extends ActorSheet {
 		data = this.computeItems(data);
 		data.carriedStates = this.#getCarriedStates();
 		data.gear = this.#filterGear(data.items);
+		if (!this.useHealthAndResolve) {
+			const filteredAttributes = Object.entries(data.system.attribute).filter(
+				([key, _]) => key !== "health" && key !== "resolve",
+			);
+			data.system.attribute = Object.fromEntries(filteredAttributes);
```

Comment:
I think it is entirely OK that the "health" and "resolve" attributes exist in the data passed to the sheet even if they are not used. So all you need is the `data.system.useHealthAndResolve = <boolean>`